docs/package.json

```
@@ -10,6 +10,7 @@
     "start": "next start"
   },
   "dependencies": {
+    "es-hangul": "workspace:^",
```

https://github.com/TanStack/query/blob/main/packages/react-query/package.json#L56

내부 패키지의 현재 버전을 의존하는 경우에 아래와 같은 방식이면 어떨까 싶은데 참고 바라요! 참조할 패키지의 같은 버전을 사용하는 것이 필요한 상황이라고 생각해서요. (제안코드를 받아드리시면 yarn install이 필요해요)

https://yarnpkg.com/features/workspaces#cross-references

```suggestion
    "es-hangul": "workspace:*",
```

---

더 정확한 방법이네요! 반영할게요!! 감사합니다~